src/metadata_client.ts

```diff
@@ -50,6 +50,25 @@ export class MetadataClient {
   ) {
     this.tagSegments = [];
     this.customTagSegments = [];
+    this.createTagSegments();
+    this.lruTag = new LruCache();
+    this.lruTagValue = new LruCache();
+    this.keyLru = new LruCache();
```

Comment:
nit; want to keep the naming consistent with the other variables? `lruKey`

Reply:
👍